tests/plugins/test_amp_plugins.py

```diff
@@ -232,3 +232,39 @@ def test_cpu_amp_precision_16_throws_error(tmpdir):
             default_root_dir=tmpdir,
             precision=16,
         )
+
+
+class GradientUnscaleNativeAMPPlugin(NativeMixedPrecisionPlugin):
+    _WAS_SCALED_NON_NAN = 0
+
+    def post_backward(
+        self, model: "pl.LightningModule", closure_loss: torch.Tensor, optimizer: torch.optim.Optimizer
+    ) -> torch.Tensor:
+        norm_before = torch.nn.utils.clip_grad_norm_(model.parameters(), 2)
+        ret_val = super().post_backward(model, closure_loss, optimizer)
+        norm_after = torch.nn.utils.clip_grad_norm_(model.parameters(), 2)
+
+        # norm_after unscale should be smaller by scaling factor greater than 1
+        if not (torch.isinf(norm_before) or torch.isnan(norm_before)):
+            assert norm_after < norm_before
```

Comment:
The scale should be much larger than 1. 

```suggestion
            assert norm_after < norm_before * 10
```

Reply:
I don't fully understand, adding the `* 10` on the right hand side of the inequality is not going to add any more value to the assertion

Reply:
The amp scaling value is quite large. Could it be possible the norm after is smaller due to un-precision `norm_after < norm_before`.

The *10 would just prevent this type of regression. This is optional.